script/Arcueid/Day 8/08_00_ARC08_1.txt

```diff
@@ -296,12 +296,12 @@ If I want to protect Akiha and those close to me, I have no choice but to lend a
 [sha:b9ede904f4df8fe370abb6749d5bfe19ff6154bc]{
 -- Page 19, Offset 19722.
 -- 気分を一転して、勢いよくベッドから起き上がる。
-I turn over my feelings and vigorously get up from the bed.
+My feelings do a complete 180 and I get out of bed with vigor.
```

Comment:
'My feelings do a complete 180' is too passive. It implies that something else caused his feelings to change rather than him putting a conscious effort to do so. With that said, I don't have a good replacement to offer

Reply:
I originally had this as "I do a complete reversal of my feelings" but that sounds a bit awkward imo

Alternatively maybe something like "I switch gears completely, psyching my self up" or  but i don't know if that would be too much of a liberty

thoughts?

Reply:
> I switch gears completely, psyching my self up

is fine but I don't necessarily agree with Yusuke's initial criticism. Your feelings doing a complete 180 does not imply you had nothing to do with it since they're like... the same entity as you.